pkg/apis/kubeonecluster/v1beta1/credentials.go

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
is it intended that this file end-up in `pkg/apis/kubeonecluster/v1beta1` instead of `pkg/apis/kubeonecluster`?

Reply:
This file will be moved